src/main/scala/io/iohk/ethereum/db/storage/FastSyncNodeStorage.scala

```diff
@@ -11,21 +11,34 @@ import io.iohk.ethereum.db.storage.encoding._
   * node saved will have its reference count equal to 1.
   *
   * */
-class FastSyncNodeStorage(nodeStorage: NodesStorage, bn: BigInt) extends ReferenceCountNodeStorage(nodeStorage, bn) {
+class FastSyncNodeStorage(cachedNodeStorage: CachedNodeStorage, bn: BigInt) extends ReferenceCountNodeStorage(cachedNodeStorage, bn) {
```

Comment:
FastSync does not have concept of caching it need to save data into database directly.